test/Org.test.js

```diff
@@ -34,79 +56,190 @@ describe("Org", function() {
     const org = await Org.new(123, this.endaomentAdmin.address, { from: admin });
     assert.isDefined(org.address);
   });
-  
+
   it("allows ORG_FACTORY to construct", async function() {
     const org = await Org.new(123, this.endaomentAdmin.address, { from: orgFactory });
     assert.isDefined(org.address);
   });
-  
+
+  it("constructs with empty active claim", async function () {
+    const org = await Org.new(123, this.endaomentAdmin.address, { from: orgFactory });
```

Comment:
123 is not a valid EIN, so this will fail when merged with #74. Using 999999999 or any other 9 digit number should work (there's a few places to change this)

Reply:
Should be easy enough to do a find/replace. Should probably pull it into a constant variable at the top of the test anyway. @mds1 mind working that into your updates?

Reply:
Yea there's a constant called `ein` in #74 set to 999999999. Can probably do the same here, or just set it to a valid number and we can update during the rebase/merge